mojo/stdlib/test/collections/test_list.mojo

```diff
@@ -294,26 +294,26 @@ def test_list_reverse_move_count():
     vec.append(MoveCounter(5))
 
     assert_equal(len(vec), 5)
-    assert_equal(vec.data[0].value, 1)
-    assert_equal(vec.data[1].value, 2)
-    assert_equal(vec.data[2].value, 3)
-    assert_equal(vec.data[3].value, 4)
-    assert_equal(vec.data[4].value, 5)
-
-    assert_equal(vec.data[0].move_count, 1)
-    assert_equal(vec.data[1].move_count, 1)
-    assert_equal(vec.data[2].move_count, 1)
-    assert_equal(vec.data[3].move_count, 1)
-    assert_equal(vec.data[4].move_count, 1)
+    assert_equal(vec[0].value, 1)
```

Comment:
FTR, the equivalent spelling should be `vec.unsafe_get(i)`; though I think ordinary indexing is also fine.